tests/decostand-tests.R

```diff
@@ -53,35 +92,52 @@ test2 <- test2[, -1]
 # Expect high correlation
 cor(unlist(test), unlist(test2)) > 0.99
```

Comment:
The comment above suggests that we should see some `TRUE` values here but in my testing, all of these correlations are *not* $>0.99$. So I'm not sure what this section is supposed to be testing. It would be good to have an actual expectation to compare with, but clarifying whether something needs to be changed here would be a start?

Reply:
This test was related to the earlier non-imputation version and can be removed.

Reply:
Great - I'll make that change locally.